.travis.yml

```diff
@@ -1,4 +1,9 @@
 language: node_js
+before_install:
+  - wget https://s3.amazonaws.com/travis-phantomjs/phantomjs-2.0.0-ubuntu-12.04.tar.bz2
```

Comment:
Totally optional idea, but you could replace the phantom...bz2 string with a variable to make it easier to switch between versions.
http://docs.travis-ci.com/user/environment-variables/